Source/WTF/wtf/SystemFree.h

```diff
@@ -31,7 +31,7 @@ namespace WTF {
 
 template<typename T>
 struct SystemFree {
-    static_assert(std::is_trivially_destructible<T>::value);
+    static_assert(std::is_trivially_destructible<T>::value || std::is_void_v<T>);
```

Comment:
Since we’re touching this line, I would use `_v` for the old part too.
```suggestion
    static_assert(std::is_trivially_destructible_v<T> || std::is_void_v<T>);
```